source/_survey-reminder.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if !current_page.data.no_survey_reminder %>
+    <div class="survey-reminder-wrapper">
+      <div class="survey-reminder-container">
+        Take the <strong>
+        <% link_to 'https://embersurvey.typeform.com/to/tKfRLF' do %>
```

Comment:
This link should go to `'/ember-community-survey-2018'`

Similar to last year's PR here: https://github.com/emberjs/website/pull/2822/files#diff-54692326d4e8adb021dbd8f90ba70f64R5